src/ImmutableX.ts

```diff
@@ -316,90 +411,137 @@ export class ImmutableX {
     orderBy?: string,
     direction?: string,
   ) {
-    return this.workflows.getProjects(
-      ethSigner,
-      pageSize,
-      cursor,
-      orderBy,
-      direction,
-    );
+    return this.workflows
+      .getProjects(ethSigner, pageSize, cursor, orderBy, direction)
+      .then(res => res.data)
+      .catch(err => {
+        throw formatError(err);
+      });
   }
 
   /**
    * Get the token Balances of the User
    * @param request - the request object containing the parameters to be provided in the API request
-   * @returns the requested Balance
+   * @returns a promise that resolves with the requested Balance
```

Comment:
this return type description is now inconsistent with other methods, probably worth reverting or changing everywhere

Reply:
I prefer the prefixed promise mention, it adds better context. have updated all other methods.

Reply:
https://github.com/immutable/imx-core-sdk/pull/172/commits/958ecc16a2bd2af4c7fd2bfd9defda71736e63d2